src/views/ReservationSuccessPage.vue

```diff
@@ -1,49 +1,75 @@
-const selectedReservation = ref();
+import { computed } from "vue";
+import { useRouter } from "vue-router";
+import { useBookingStore } from "@/stores/booking";
+import { useMovieStore } from "@/stores/movies";
+import { useUserStore } from "@/stores/user";
+import { useTitle } from "@vueuse/core";
+import { getTicketById } from "@/api/ticketID"
+import ReservationItem from "@/components/reservations/ReservationItem.vue";
+import BaseButton from "@/components/base/BaseButton.vue";
+const router = useRouter()
+const bookingStore = useBookingStore();
+const movieStore = useMovieStore()
+const userStore = useUserStore()
+useTitle("Moneterail | Your reservation");
 
-watch(queryReservation, async (newVal) => {
-  console.log(newVal);
-  // const email = parseInt(newVal);
-  // const response = await getSingleReservation(email);
-  // selectedReservation.value = response;
-});
+const selectedMovie = computed(() => {
+  return movieStore.movieById(bookingStore.selectedReservation.movie) || {title: ''}
+})
+const reservationEmail = computed(() => {
+  return bookingStore.customerEmail ? bookingStore.customerEmail : userStore.user.email
+})
+const bookedTickets = computed(() => {
+  return bookingStore.selectedTickets.map((ticket) => {
+    return {
+      ...ticket,
+      id: ticket.seat,
+      status: 'Confirmed',
+      datetime: bookingStore.selectedReservation.datetime,
+      movie: selectedMovie.value.title,
+      email: reservationEmail.value,
+      price: getTicketById(ticket.ticket_type_id)?.price || '',
+      type: getTicketById(ticket.ticket_type_id)?.type || '',
+      }
+  })
+})
+const ctaText = computed(() => {
+  return userStore.isEmployee ? 'Go to ticket desk' : 'Go to homepage'
+})
 
-// const bookedTickets = computed(() => {
-//   return bookingStore.selectedTickets.map((ticket) => {
-//     return {
-//       ...ticket,
-//       movie: bookingStore.selectedReservation.movie,
-//       datetime: bookingStore.selectedReservation.datetime,
-//     };
-// //   });
-// });
+const handleCta = () => {
+  userStore.isEmployee ? router.push({ name: 'EmployeePage'}) : router.push({ name: 'HomePage'})
+}
 </script>
 
 <template>
   <div class="success">
     <h1 class="success__title">
       Hell yeah! <br />
-      <span class="success__title--red">You booked {{ 3 }} tickets</span>
+      <span class="success__title--red">You booked {{ bookedTickets.length }} tickets</span>
     </h1>
-    <div class="sucess__tickets">
-      <!-- <ReservationItem -->
-      <!--   v-for="ticket in bookedTickets" -->
-      <!--   :key="ticket.seat" -->
-      <!--   :ticket="ticket" -->
-      <!-- /> -->
+    <div v-if="bookedTickets" class="sucess__tickets">
+      <ReservationItem
+        v-for="ticket in bookedTickets"
+        :key="ticket.seat"
+        :ticket="ticket"
+      />
+    </div>
+    <div class="success__buttons">
+      <BaseButton size="medium" @click="handleCta">{{ ctaText }}</BaseButton>
```

Comment:
While this is working properly, we could technically just make a computed (something like `ctaLocation`) and bind it to the `to` prop - this way we have a link when it should be a link 😄 

```suggestion
      <BaseButton size="medium" :to="ctaLocation">{{ ctaText }}</BaseButton>
```